card/mod/pointer/lib/javascript/script_pointer_config.js.coffee

```diff
@@ -11,10 +11,13 @@ $.extend decko.editorContentFunctionMap,
       pointerContent @find('input:checked').map( -> $(this).val() )
     '.pointer-select-list': ->
       pointerContent @find('.pointer-select select').map( -> $(this).val() )
-    '.pointer-mixed': ->
-      element = '.pointer-checkbox-sublist input:checked,\
-                .pointer-sublist-ul input'
-      pointerContent @find(element).map( -> $(this).val() )
+    '.pointer-filtered-list': ->
+      pointerContent @find('.filtered-list-item').map( -> $(this).data('cardname') )
+    # can't find evidence that the following is in use: #efm
+    # '.pointer-mixed': ->
+    #   element = '.pointer-checkbox-sublist input:checked,\
+    #             .pointer-sublist-ul input'
+    #   pointerContent @find(element).map( -> $(this).val() )
```

Comment:
:alert: I commented code that I am only 95% confident is not used.

Reply:
I'm never 100% sure of anything.

Reply:
But I was hoping you would help me chip away at the other 5%.  I've looked in wikirate and decko for `.pointer-mixed` (and even inspected every case of `mixed` I can find).  Is there anything else I should do?

Reply:
+3% check wikirate's database
+1% make sure all mods where included in your search path
+0.5% check decko's seed data
+0.5% check all decks we have code for
+1% use git's pickaxe to figure out where and why it was introduced (It was used for the edit interface in right/following.rb)

Reply:
is 101% confidence the new threshold?

Reply:
not in wikirate, not in mods, not in seed data (except for various places where the commented line is pulled in).  The only other decko code I could think of was wagn/wagn-cldstr, and it's not referenced there. I gather you looked in the pickaxe already?

Reply:
no, still 100%. If you find the place in the code where it was used and a reason why it's no longer there I won't insist that you search the seed data and also the other way round: If you really checked all places it doesn't seem necessary to investigate the history.

Reply:
it's not in decko's seed data. (it's only in wikirate seed data, and, as I said, it's only the same line of javascript that I mentioned here.

Here's the commit where you removed `pointer-mixed`:  https://github.com/decko-commons/decko/commit/c94c6f53190bc10e17feb3ea9b9367de734082ee  It doesn't say exactly why, but I can't see any references to list_second, which you removed at the time.

(to see the diff you have to expand mod/05_email/set/right/following.rb).

So, it's a js function that was triggered by the the presence of a class that no longer exists. It was last present in a view that no longer exists defined within a file that no longer exists.  I think this thing is pretty much dead.

Reply:
I'll take that as 100% :smile: